src/App.jsx

```diff
@@ -14,33 +15,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [list, setList] = useState(goodsFromServer);
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortedState, setSortedState] = useState(0);
```

Comment:
```suggestion
  const [sortType, setSortType] = useState(ʼʼ);
```
better to save the strings here, so you can save `alphabetically` or `length` or empty string